xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -56,7 +56,7 @@ To be able to format the output with indentation and white space OR to skip vali
       <Docs>
         <param name="bufferWriter">The destination for writing JSON text.</param>
         <param name="options">Defines the customized behavior of the <see cref="T:System.Text.Json.Utf8JsonWriter" /> By default, it writes minimized JSON (with no extra whitespace) and validates that the JSON being written is structurally valid according to the JSON RFC.</param>
```

Comment:
```suggestion
        <param name="options">One of the enumeration values that defines the customized behavior of the <see cref="T:System.Text.Json.Utf8JsonWriter" /> By default, it writes minimized JSON (with no extra white space) and validates that the JSON being written is structurally valid according to the JSON RFC.</param>
```

Reply:
Adding missing dot to @mairaw's suggestion.
```suggestion
        <param name="options">One of the enumeration values that defines the customized behavior of the <see cref="T:System.Text.Json.Utf8JsonWriter" />. By default, it writes minimized JSON (with no extra white space) and validates that the JSON being written is structurally valid according to the JSON RFC.</param>

```

Reply:
The options isn't an enum though.